Sources/Quick/Example.swift

```diff
@@ -136,6 +119,65 @@ final public class Example: _ExampleBase {
         }
         return aggregateFlags
     }
+	
+	private func reportSkippedTest(_ testSkippedError: XCTSkip) {
+		#if _runtime(_Native)
+			return // This functionality is Only supported by Apple's proprietary XCTest, not by swift-corelibs-xctest
+		#elseif _runtime(_ObjC)
```

Comment:
What's the correct way check that we're using Apple's XCTest and not [corelibs-xctest](https://github.com/apple/swift-corelibs-xctest)? I see several options:

* `#if canImport(Darwin)`
* `#if _runtime(_ObjC)`
* `#if #if os(macOS) || os(iOS) || os(tvOS)`, seems fragile, it'll omit future targets that may be supported.

I also asked about this on the Swift forums: https://forums.swift.org/t/whats-the-right-way-to-distinguish-between-corelibs-frameworks-from-apple-propreitary-ones/51699

Reply:
Let's use `#if canImport(Darwin)`.

- https://github.com/Quick/Quick/pull/1040#discussion_r711749726
- #1104